### ISSUE TITLE
swupd update should repeat incremental updates until latest

### ISSUE THREAD

**Issue by puneetse** [clearlinux/swupd-client#1197]
**Describe the bug**
I booted an old system (30130) and did `swupd update`. It only took me up to 30760 and stopped, presumably because of a format bump. 

**To Reproduce**

- Start with an older Clear Linux OS version over a format bump.
- Do a `swupd update`. 
- First set of update starts 30130-to-30760

**Expected behavior**
Invoking `swupd update` again kept going but I'd expect swupd to continue repeating itself until it is done before exiting.

I also got these going from 30770 to version 31520. @phmccarty 

```
Warning: Couldn't use delta file /var/lib/swupd/delta/30770-31520-ddedcff6f7477ee1ace6c4e316513ce188df4deea193dc24b0390cecb74ac44b-8dbc61bd38ad9fb48461751c279a10221cf03e8e9647948ac1ea8539a5d2ef83: 'from' file corrupted on system, consider running 'swupd verify --fix'
Warning: Couldn't use delta file /var/lib/swupd/delta/30770-31520-5d0dfb8b29db6189dbee5634300712861f1755becea04bc86e4a9a8d6e6d4ae5-22d0e96c0db1bf4563fab07f922168bf987edb8cf255da15704d9620b7a2d1bf: 'from' file corrupted on system, consider running 'swupd verify --fix'
Warning: Couldn't use delta file /var/lib/swupd/delta/30770-31520-b325c70413dc7d6d686f340fa420ff12408f755d17bfe3a6f06b4238230b87f1-af5b0a16bddffa48f157709f79a4e0fc6cad386feaaa12b940fbc0f7de986248: 'from' file corrupted on system, consider running 'swupd verify --fix'
Warning: Couldn't use delta file /var/lib/swupd/delta/30770-31520-d813b1deca79200b74ab8fd1ab82fe77411bf8eb9ea4930fd4672c593a9b3582-7899c74efd95a475ec63944bb3b08ec14daad633676bcbf5365f616f3184152a: 'from' file corrupted on system, consider running 'swupd verify --fix'
Warning: Couldn't use delta file /var/lib/swupd/delta/30770-31360-0103bf7b5d4fd8e8d52534cee033f31718ec079e5c5aebd229e53e9d07ce8fd5-0fd524cb02820b42c37073c27108242d73ffe5bc1d28d9e26e188f3fad374e1e: 'from' file corrupted on system, consider running 'swupd verify --fix'
```

**Comment by phmccarty** [comment]
@puneetse I tested those five deltas, and they are all valid. All of those files live under `/usr/lib/modules/*.native/` and were from the `linux` package. Did you have any custom native kernels installed on that system that may have had the same NVRs (name-version-release) as the ones in the distro?

Regarding the re-exec issue, I think #882 had resolved it, but I'm not sure about when it was merged...
